eth/protocols/eth/handlers.go

```diff
@@ -352,8 +385,11 @@ func handleBlockBodies(backend Backend, msg Decoder, peer *Peer) error {
 			if body.Withdrawals != nil {
 				withdrawalHashes[i] = types.DeriveSha(types.Withdrawals(body.Withdrawals), hasher)
 			}
+			if body.Sidecars != nil {
```

Comment:
I think the `sidecarsHashes` is useless, it cannot give help any validation. It can be removed. I think just modify `DeliverBodies`, and add blobs for `fetchResult` in `reconstruct`.

The `importBlockResults` in `Downloader` also need modify to append blobs.
```go
for i, result := range results {
		blocks[i] = types.NewBlockWithHeader(result.Header).WithBody(result.Transactions, result.Uncles).WithWithdrawals(result.Withdrawals).WithBlobs(result.Blobs)
	}
```